app-imaging/glslang/autobuild/defines

```diff
@@ -1,7 +1,10 @@
 PKGNAME=glslang
 PKGSEC=graphics
-PKGDEP="python-2 glibc"
-BUILDDEP="bison"
+PKGDEP="python-3 glibc"
```

Comment:
`python-3` should probably be in BUILDDEP instead, there is no Python component nor ELF/Shebang dependency on it.

Reply:
Done.

Reply:
> Done.

I don't see that you've pushed any changes?